packages/external/motion/package.json

```diff
@@ -1,11 +1,12 @@
 {
-  "name": "@heroui/framer-utils",
-  "version": "2.1.7",
-  "description": "A set of framer motion transitions for react",
+  "name": "@heroui/motion",
+  "version": "2.0.0",
+  "description": "Mirror package of framer-motion",
   "keywords": [
-    "framer-utils"
+    "hero ui",
+    "motion"
   ],
-  "author": "Junior Garcia <jrgarciadev@gmail.com>",
+  "author": "WK Wong <wingkwong.code@gmail.com>",
```

Comment:
@wingkwong let's refactor all packages' authorm add "HeroUI <support@heroui.com>"

Reply:
sure. will update this one and handle the existing components in another PR.